api/envoy/config/core/v3/base.proto

```diff
@@ -237,6 +237,13 @@ message Metadata {
   // Key is the reverse DNS filter name, e.g. com.acme.widget. The envoy.*
   // namespace is reserved for Envoy's built-in filters.
   map<string, google.protobuf.Struct> filter_metadata = 1;
+
+  // Key is the reverse DNS filter name, e.g. com.acme.widget. The envoy.*
+  // namespace is reserved for Envoy's built-in filters.
+  // The value is encoded as google.protobuf.Any.
+  // If both filter_metadata and typed_filter_metadata fields are present in the
```

Comment:
Please add the same comment to `filter_metadata`.

Reply:
Also, use `*typed_filter_metadata*`, `*filter_metadata*` formatting (or even RST links) to indicate these are field names.

Reply:
Thanks, Done!